### DISCUSSION TITLE
Coupled Input

### DISCUSSION THREAD

**Discussion by FelixBenning** [fonsp/Pluto.jl#1291] [Questions about using Pluto]
I was just thinking how I would parametrize the selection of eigenspaces for visualizing a quadratic function in 2 dimension. In Taylor approximations the Hesse Matrix is symmetric so I thought I would restrict myself to such symmetric matrices too. But that restricts eigenvectors to be orthogonal.

Is there a way to allow the selection of both eigenvectors in two dimensions (e.g. an angle slider), where the other one updates if you modify the first?

**Comment by tjhhunter** [comment]
I'm still a neophyte, so I hesitate a bit to speak up, but does the following do the kind of thing you want? (In two different cells.)

``@bind x Slider(1:10)``

``@bind y Slider(1:20,default=2x)``

**Reply by FelixBenning** [reply]
This would (maybe[*]) update `y` whenever `x` is changed but definitely not update `x` when `y` is changed. 

[*]: I am not sure if cells are rerun if they depend on a different cell only for a default. I would say probably yes because pluto is probably not clever enough to realize that the dependency is gone once someone moved the slider away from the default, but that might be false.
